containers/java-postgres/test-project/src/main/java/com/mycompany/app/App.java

```diff
@@ -0,0 +1,34 @@
+/*-------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import static javax.xml.XMLConstants.XML_NS_PREFIX;
+import java.io.*;
```

Comment:
nit: Avoid using *. Import specific classes only.

Reply:
I'll change this, good catch